packages/store/addon/-private/system/record-arrays/record-array.js

```diff
@@ -113,8 +124,13 @@ export default ArrayProxy.extend(DeprecatedEvented, {
     @return {Model} record
   */
   objectAtContent(index) {
-    let internalModel = get(this, 'content').objectAt(index);
-    return internalModel && internalModel.getRecord();
+    let identifier = get(this, 'content').objectAt(index);
```

Comment:
this might read nicer if 
```
    if (RECORD_ARRAY_MANAGER_IDENTIFIERS) {
        let identifier = get(this, 'content').objectAt(index);
      return identifier ? recordForIdentifier(this.store, identifier) : undefined;
    } else {
      return identifier ? identifier.getRecord() : undefined;
    }
```